internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -835,10 +837,14 @@ func (r *ClusterDeploymentsReconciler) updateNetworkParams(clusterDeployment *hi
 		}
 	}
 
-	if clusterInstall.Spec.Networking.NetworkType != "" {
+	if swag.StringValue(cluster.Status) != "installed" && clusterInstall.Spec.Networking.NetworkType != "" {
```

Comment:
you can use
https://github.com/openshift/assisted-service/blob/1f7410a2d43220908c4ef1b136d2075ac284ec96/models/cluster.go#L1077